charts/openshift-etcd-backup/templates/cronjob.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "openshift-etcd-backup.fullname" . }}
+spec:
+  schedule: {{ .Values.backup.schedule | quote }}
+  jobTemplate:
+    spec:
+      backoffLimit: 0
```

Comment:
right, i'm not sure about the idea behind that tbh.
cyru might know more, but it's risky and we could remove that i think!

Reply:
since im getting notifyed and annoyed by this tread i can answer as well: yes this is intended. if the script fails, it will fail the 2nd time as well normally. since it is writing and deleting stuff on an volume, just retrying can be dangerous. if it is failing, someone has to check whats gone wrong, that was the idea behind.

btw: wont answer again, unsubscribed for now.

Reply:
Thanks Cyru! 

> since im getting notifyed and annoyed by this tread i can answer as well

That's on you, fix your notification settings.

Reply:
haha sorry for that @eni23 and thanks for the explanation!

Reply:
@eni23 That was my expectation why it is configured that way. Thanks for confirming. :)